src/main/features/response/form/models/howMuchOwed.ts

```diff
@@ -0,0 +1,42 @@
+import { Serializable } from 'models/serializable'
+import { IsDefined, IsPositive, MaxLength } from 'class-validator'
+import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
+
+export class ValidationErrors {
+  static readonly NOT_OWE_FULL_AMOUNT_REQUIRED: string = 'Explain why you don’t owe the full amount'
+  static readonly REASON_NOT_OWE_MONEY_TOO_LONG: string = 'Enter reason no longer than $constraint1 characters'
+  static readonly VALID_OWED_AMOUNT_REQUIRED: string = 'Enter a valid amount owed'
+  static readonly AMOUNT_REQUIRED: string = 'Enter an amount'
+}
+
+export class HowMuchOwed implements Serializable<HowMuchOwed> {
+
+  @IsDefined({ message: ValidationErrors.AMOUNT_REQUIRED })
+  @IsPositive({ message: ValidationErrors.VALID_OWED_AMOUNT_REQUIRED })
+  amount?: number
+  @IsDefined({ message: ValidationErrors.NOT_OWE_FULL_AMOUNT_REQUIRED })
+  @IsNotBlank({ message: ValidationErrors.NOT_OWE_FULL_AMOUNT_REQUIRED })
+  @MaxLength(99000, { message: ValidationErrors.REASON_NOT_OWE_MONEY_TOO_LONG })
```

Comment:
can you please add ValidationConstatnts class to keep all consts together (you can reuse it in tests then)

Reply:
Is it not a good idea to reuse existing validation annotations?

Reply:
I think we can extract ValidationConstants from one of the files it already exist to some common module and reuse it here. 

I'd rather not use class from other local module as it creates weird dependency.

Reply:
Ok fair good point